### DISCUSSION TITLE
A problem about the GUI page

### DISCUSSION THREAD

**Discussion by liuwenrui917** [MannLabs/alphapeptdeep#184] [General]
Hello Dr. Zeng
Thank you for your reply. And I want to ask you that I will use High-pH Reversed-Phase Fractionation to build a library with Fragpipe 22. But when I open pepdeep GUI，it appears an error like this. Do you know how to solve this problem?
Thanks very much.
![fragpipe](https://github.com/user-attachments/assets/7f755921-142b-4684-8be4-d5300eabcb19)

**Comment by jalew188** [comment]
Working in progress #186
